src/style.css

```diff
@@ -1 +1,27 @@
 /* styles go here */
+.personal,
+.register,
+.aboutYou,
+.additInfo {
+  display: block;
+}
+
+.group:not(:last-child) {
+  margin-bottom: 20px;
+}
+
+.personal:not(:last-child) {
+  margin-bottom: 10px;
+}
+
+.register:not(:last-child) {
+  margin-bottom: 10px;
+}
+
+.aboutYou:not(:last-child) {
+  margin-bottom: 10px;
+}
+
+.additInfo:not(:last-child) {
+  margin-bottom: 10px;
+}
```

Comment:
no needs to add all of this classes. 
You may only apply those to your `filedset` and form field (`label` or `div`)
Example: 
```
.fieldset:not(:last-child) {
  margin-bottom: 20px
}

.field:not(:last-child) {
  margin-bottom: 10px
}
```